### PR TITLE
Feature/cbi 720 fix test after burn remove

### PR DESCRIPTION
**Pull request by AndreiNavoichyk**

### CODE REVIEW

File: contracts/ERC20Safe.sol [hunk 1]
```diff
@@ -2,6 +2,7 @@ pragma solidity >=0.6.0 <0.8.0;
 
 import "@openzeppelin/contracts/math/SafeMath.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/presets/ERC20PresetMinterPauser.sol";
```
Comment (AndreiNavoichyk) [inline contracts/ERC20Safe.sol:5]: Why do we need it here?

File: test/contractBridge/admin.js [hunk 2]
```diff
@@ -138,14 +138,14 @@ contract('Bridge - [admin]', async accounts => {
 
     // Set burnable
 
-    it('Should set ERC20MintableInstance.address as burnable', async () => {
-        const ERC20MintableInstance = await ERC20MintableContract.new("token", "TOK");
-        const resourceID = Helpers.createResourceID(ERC20MintableInstance.address, chainID);
-        const ERC20HandlerInstance = await ERC20HandlerContract.new(BridgeInstance.address, [resourceID], [ERC20MintableInstance.address], []);
+    // it('Should set ERC20MintableInstance.address as burnable', async () => {
```
Comment (AndreiNavoichyk) [inline test/contractBridge/admin.js]: Let's remove it and not comment

File: test/e2e/erc721/differentChainsMock.js [hunk 3]
```diff
@@ -1,207 +0,0 @@
-const TruffleAssert = require('truffle-assertions');
```
Comment (AndreiNavoichyk) [inline test/e2e/erc721/differentChainsMock.js]: Let's revert it back. We don't use ERC721 but still have it in the SCs.

File: test/e2e/erc721/sameChain.js [hunk 4]
```diff
@@ -1,126 +0,0 @@
-const TruffleAssert = require('truffle-assertions');
```
Comment (AndreiNavoichyk) [inline test/e2e/erc721/sameChain.js]: Let's revert it. We don't use ERC721 but still have in the repo.

File: test/gasBenchmarks/deposits.js [hunk 5]
```diff
@@ -2,232 +2,232 @@
...
-
-        gasBenchmarks.push({
-            type: 'Generic - Three Arguments',
-            gasUsed: depositTx.receipt.gasUsed
-        });
-    });
-
-    it('Should print out benchmarks', () => console.table(gasBenchmarks));
-});
+ const BridgeContract = artifacts.require("Bridge");
```
Comment (AndreiNavoichyk) [inline test/gasBenchmarks/deposits.js]: @karthikeyan-cere 
You should avoid such changes like add 1 space in each line in the file. Better not even commit it locally but revert and add only relevant changes.

File: test/gasBenchmarks/executeProposal.js [hunk 6]
```diff
@@ -2,262 +2,263 @@
...
-
-        gasBenchmarks.push({
-            type: 'Generic - Three Argument',
-            gasUsed: executeTx.receipt.gasUsed
-        });
-    });
-    
-    it('Should print out benchmarks', () => console.table(gasBenchmarks));
-});
+ const Ethers = require('ethers');
```
Comment (AndreiNavoichyk) [inline test/gasBenchmarks/executeProposal.js]: What has been changed here?

File: test/handlers/erc721/deposit.js [hunk 7]
```diff
@@ -1,116 +0,0 @@
-/**
```
Comment (AndreiNavoichyk) [inline test/handlers/erc721/deposit.js]: Revert it back. We don't use ERC721 but it is still in the repo.

File: test/contractBridge/admin.js (hunk 2 above)
Comment (karthikeyan-cere) [reply test/contractBridge/admin.js]: Done

File: test/e2e/erc721/differentChainsMock.js (hunk 3 above)
Comment (karthikeyan-cere) [reply test/e2e/erc721/differentChainsMock.js]: Reverted the changes

File: test/e2e/erc721/sameChain.js (hunk 4 above)
Comment (karthikeyan-cere) [reply test/e2e/erc721/sameChain.js]: Reverted the changes

File: test/handlers/erc721/deposit.js (hunk 7 above)
Comment (karthikeyan-cere) [reply test/handlers/erc721/deposit.js]: Reverted the changes

File: test/gasBenchmarks/deposits.js (hunk 5 above)
Comment (karthikeyan-cere) [reply test/gasBenchmarks/deposits.js]: Reverted the changes

File: test/gasBenchmarks/executeProposal.js (hunk 6 above)
Comment (karthikeyan-cere) [reply test/gasBenchmarks/executeProposal.js]: Reverted changes

File: contracts/ERC20Safe.sol (hunk 1 above)
Comment (karthikeyan-cere) [reply contracts/ERC20Safe.sol:5]: We need to include this else it will not be included in the smart contract. and it will through error while testing it.

**Pull request closed without merging**